docs/getting-started/form-submissions.md

```diff
@@ -71,6 +72,42 @@ query set as the `q` query parameter in the URL. The `GET` handler will then
 filter the names array based on the query, and pass it to the page component for
 rendering.
 
+## `POST` example
+Here's an example processing a simple form submission request:
```

Comment:
simple is not a good word to use in technical documentation: https://developers.google.com/style/word-list#simple

Reply:
Agreed. Old habits die hard.